gpytorch/__init__.py

```diff
@@ -193,3 +227,43 @@ def test_train_mul_closure(v):
         test_test_covar = test_test_covar.sub(test_test_covar_correction)
 
     return GaussianRandomVariable(test_mean, test_test_covar), alpha
+
+
+def _variational_predict(n, full_covar, variational_mean, chol_variational_covar, alpha=None):
+    test_train_covar = full_covar[n:, :n]
+    train_test_covar = full_covar[:n, n:]
+    if isinstance(test_train_covar, LazyVariable):
+        test_covar = chol_variational_covar.t().mm(chol_variational_covar)
+        add_jitter(test_covar)
+        test_covar = train_test_covar.apply_right_interpolation(test_covar)
+        test_covar = test_train_covar.apply_left_interpolation(test_covar)
+
+        test_mean = test_train_covar.apply_left_interpolation(variational_mean.unsqueeze(1)).squeeze()
```

Comment:
We can wrap this in a Function. I was about to refactor the `_exact` stuff. Let's talk about it later.